rhel/vex/updater.go

```diff
@@ -11,15 +11,21 @@ import (
 
 	"github.com/quay/zlog"
 
+	"github.com/quay/claircore"
+
 	"github.com/quay/claircore/libvuln/driver"
 )
 
 var (
-	_ driver.Updater           = (*Updater)(nil)
-	_ driver.Configurable      = (*Updater)(nil)
-	_ driver.DeltaUpdater      = (*Updater)(nil)
-	_ driver.UpdaterSetFactory = (*Factory)(nil)
-	_ driver.Configurable      = (*Factory)(nil)
+	_        driver.Updater           = (*Updater)(nil)
+	_        driver.Configurable      = (*Updater)(nil)
+	_        driver.DeltaUpdater      = (*Updater)(nil)
+	_        driver.UpdaterSetFactory = (*Factory)(nil)
+	_        driver.Configurable      = (*Factory)(nil)
+	rhccRepo                          = &claircore.Repository{
```

Comment:
nit: put a newline above this, so the lines above this won't need to be modified

Reply:
Also doesn't the `goldRepo` already exist?